app/scripts/models/reliers/oauth.js

```diff
@@ -204,6 +207,15 @@ define(function (require, exports, module) {
     },
 
     /**
+     * Check if the relier wants access to the account encryption keys.
+     *
+     * @returns {Boolean}
+     */
+    wantsKeys () {
+      return !! this.has('keys_jwk');
```

Comment:
As a security measure, let's ensure that this check returns `false` when the feature is preffed off, so that we don't even open the possibility of fetching keys unexpected.

Any additional sanity-checking we can do here will be valuable I think, e.g. if we can detect that they've provided a `keys_jwk` but no key-bearing scopes, we'll be better off erroring out early here rather than fetching the keys and erroring out later on.  I don't know what's possible within the scope of the current code structure though.